stubs/RdKafka.php

```diff
@@ -81,4 +74,37 @@ public function setLogLevel($level)
     public function queryWatermarkOffsets($topic, $partition, &$low, &$high, $timeout_ms)
     {
     }
+
+    /**
+     * @param array $topicPartitions
+     * @param int   $timeout_ms
+     * @return array
+     */
+    public function offsetsForTimes (array $topicPartitions , int $timeout_ms) : array
```

Comment:
If you specify return type or primitive types in the method definition you have to specify minimum php version of 7.x in the composer.json